src/commands/stac-setup/category.constants.ts

```diff
@@ -4,6 +4,7 @@ export const dataCategories = {
   RURAL_AERIAL_PHOTOS: 'rural-aerial-photos',
   SATELLITE_IMAGERY: 'satellite-imagery',
   URBAN_AERIAL_PHOTOS: 'urban-aerial-photos',
+  RASTER_TOPO_MAPS: 'raster-topo-maps',
```

Comment:
@blacha We need add a category for the `raster-topo-map`, do you have a better name for it?

Reply:
I think `topographic-maps` was the category chosen.